### DISCUSSION TITLE
请问双击按钮但不操作全屏是如何实现的？

### DISCUSSION THREAD

**Discussion by Pavomuticus** [dyphire/mpv-config#14] [Q&A] [answered]
脚本没看明白，取一下经🙏

**Comment by Pavomuticus** [comment, marked as answer]
依靠[inputevent.lua](https://github.com/Natural-Harmonia-Gropius/InputEvent)简单配置即可避免在按钮上双击时意外触发全屏
input.conf添加`MBTN_LEFT cycle fullscreen #@double_click`

**Reply by zhongfly** [reply]
> input.conf添加`MBTN_LEFT cycle fullscreen`

这也没用上[inputevent.lua](https://github.com/Natural-Harmonia-Gropius/InputEvent)吧

**Reply by hooke007** [reply]
怎么感觉过于绕？屏蔽双击 `MBTN_LEFT_DBL ignore` 不就完事了

**Reply by dyphire** [reply]
他这个问题的本质是mpv的鼠标双击动作会触发单击动作，而`MBTN_LEFT_DBL ignore`和`input-default-bindings=no`本质都是忽略双击动作，并没有实质解决问题。[inputevent.lua](https://github.com/Natural-Harmonia-Gropius/InputEvent)提供了另一种逻辑来处理不同的触发动作从而避免此问题

**Reply by Pavomuticus** [reply]
> > input.conf添加`MBTN_LEFT cycle fullscreen`
> 
> 这也没用上[inputevent.lua](https://github.com/Natural-Harmonia-Gropius/InputEvent)吧

手快字打少了🤪

**Reply by Pavomuticus** [reply]
只要`MBTN_LEFT cycle fullscreen #@double_click`就可以达到目的，`input-default-bindings=no`只是方便自定义
